### ISSUE TITLE
saving model error

### ISSUE THREAD

**Issue by saeed-anwar** [yulunzhang/RCAN#13]
Hi,

while running your code, there is a problem with the saving model. It gives an error.

The error arises in the test section of the trainer when saving the model.

Please, can you look into the problem?

Regards,
Saeed

**Comment by saeed-anwar** [comment]
I have solved this problem, that is, when we need to put trainer.model.module.save() 
in line 79 of utility.

However, there is another problem, during training, the PSNR is 31.3 however, when testing on the same set the PSNR becomes 12.7, which means the test code is not loading the model properly.

**Comment by yulunzhang** [comment]
The problem should that your provided network structure doesn't match the trained network. 

The provided code can be run for train and test. If you run it according to the provided scripts, it would be ok. 

Could you please follow the provided instruction to run the code firstly?

**Comment by saeed-anwar** [comment]
Yes, it is working fine now. However, I have a question, do you save the images during the test and then calculate the PSNR and SSIM using the provided Matlab file.

**Comment by yulunzhang** [comment]
Yes

**Comment by saeed-anwar** [comment]
Thanks,
